pkg/internal/apis/certmanager/v1alpha2/zz_generated.conversion.go

```diff
@@ -579,12 +590,8 @@ func autoConvert_v1alpha2_CertificateSpec_To_certmanager_CertificateSpec(in *v1a
 	return nil
 }
 
-// Convert_v1alpha2_CertificateSpec_To_certmanager_CertificateSpec is an autogenerated conversion function.
-func Convert_v1alpha2_CertificateSpec_To_certmanager_CertificateSpec(in *v1alpha2.CertificateSpec, out *certmanager.CertificateSpec, s conversion.Scope) error {
-	return autoConvert_v1alpha2_CertificateSpec_To_certmanager_CertificateSpec(in, out, s)
-}
-
 func autoConvert_certmanager_CertificateSpec_To_v1alpha2_CertificateSpec(in *certmanager.CertificateSpec, out *v1alpha2.CertificateSpec, s conversion.Scope) error {
+	// WARNING: in.X509Name requires manual conversion: does not exist in peer-type
```

Comment:
These messages should go away once you update the internal type to use the field name `Subject` instead of `X509Name`